homeassistant/components/light/mqtt_json.py

```diff
@@ -108,6 +127,14 @@ def __init__(self, hass, name, topic, qos, retain,
 
         self._flash_times = flash_times
 
+        self._templates = {}
+        for key, tpl in list(templates.items()):
+            if tpl is None:
```

Comment:
Since you have this check, you can remove the default values for the config options.